test/torchaudio_unittest/common_utils/ctc_decoder_utils.py

```diff
@@ -0,0 +1,32 @@
+from torchaudio.models.decoder import CTCDecoderLM, CTCDecoderLMState
```

Comment:
I don't think these will be reused outside of `ctc_decoder_test.py`. Can we put this there for now?